util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -14,11 +14,12 @@
 
 from __future__ import annotations
 
+from collections.abc import Mapping
```

Comment:
This should fail with old python because it's subscribed

Reply:
What do you mean by subscribed?

Reply:
`Mapping[..]` like in line 91 https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2361/files#diff-9f93b3514e50dc2c60507d1deb1ad7bd80b8dcca02a5d1cb4bcec412b5cdcbaeR91

Reply:
It didn't fail any test :shrug: